dynamodb/src/main/resources/META-INF/services/org.apache.metamodel.factory.DataContextFactory

```diff
@@ -0,0 +1 @@
+org.apache.metamodel.dynamodb.DynamoDbDataContextFactory
```

Comment:
Missing EOL at end of file (ignore if it's needed like this by the serviceloader, but I don't believe it is)

Reply:
Added EOL